msm/src/serialization/witness.rs

```diff
@@ -17,8 +17,9 @@ pub struct Env<Fp> {
     /// field Kimchi gate
     pub intermediate_limbs: [Fp; N_INTERMEDIATE_LIMBS],
 
-    pub lookup_multiplicities_rangecheck4: [Fp; 1 << 4],
-    pub lookup_multiplicities_rangecheck15: [Fp; 1 << 15],
+    // Boxing to avoid stack overflow
```

Comment:
Isn't this already done in the previous PR? https://github.com/o1-labs/proof-systems/pull/1896

Reply:
Yes. I cherry-picked the commit when I saw that the code wasn't happy at runtime. I thought GH would handle this cherry-pick.